2024-09-03 13:35:46,238 | WARNING | Неверный тип данных для имени объекта Runner
Traceback (most recent call last):
  File "C:\Users\vavan\PythonProjectsUrban\unit_testing\hmw_2\tests_12_4.py", line 25, in test_run
    raise TypeError(f'Имя может быть толко строковым значением, передано <{type(some_runner.name).__name__}>')
TypeError: Имя может быть толко строковым значением, передано <int>
2024-09-03 13:35:46,238 | WARNING | Неверная скорость для Барсик
Traceback (most recent call last):
  File "C:\Users\vavan\PythonProjectsUrban\unit_testing\hmw_2\tests_12_4.py", line 15, in test_walk
    raise ValueError(f'Скорость может быть только положительным значением, текущее значение = {some_runner.speed}')
ValueError: Скорость может быть только положительным значением, текущее значение = -10
